server/data/projects.js

```diff
@@ -93,6 +93,23 @@ const createProject = async (projectObjParam, user) => {
 	);
 	return createdProject;
 };
+const likeProject = async (user, project) => {
+	const userInfo = user;
+	const userId = userInfo._id;
```

Comment:
you also need to validate the username as you are sending it into this function. Everything that comes to the function needs to be validated or don't send it if you don't need it